RHEL/6/input/checks/file_permissions_ungroupowned.xml

```diff
@@ -16,9 +15,19 @@
   <unix:file_test xmlns:unix="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix"
   check="all" comment="files with no group owner" check_existence="none_exist"
   id="test_file_permissions_ungroupowned" version="1">
+    <!-- OVAL 5.11 rc-2 within the effort to add oval-def:NotesType construct also to variables
+         moved NotesType construct to the oval-common-schema:
+         [1] https://github.com/OVALProject/Language/issues/6
+         [2] https://github.com/OVALProject/Language/commit/c34abb871ee446d9fd9e50a8ccbc8a8e2af5db20
+         But this change broke backward-compatibility as reported in:
+         [3] https://github.com/OVALProject/Language/issues/237
+         [4] http://making-security-measurable.1364806.n2.nabble.com/5-11-backwards-incompatibility-issue-Suspect-the-fix-for-issue-6-was-the-culprit-tp7586190.html
+         Therefore as a temporary workaround to have both OVAL 5.10 and also OVAL 5.11 valid content,
+         temporarily comment out the <notes> construct below till the issue [3] is fixed in OVAL 5.11.1
+         version.
     <notes>
       <note>This will enumerate all files on local partitions</note>
-    </notes>
+    </notes> -->
```

Comment:
Yes, it's intentional. Because when not commented out, the following error message is printed out:
```
File '/root/scap-security-guide/rpmbuild/BUILD/scap-security-guide-0.1.21/RHEL/6/output/ssg-rhel6-ds.xml' line 9487: Element '{http://oval.mitre.org/XMLSchema/oval-definitions-5}note': This element is not expected. Expected is ( {http://oval.mitre.org/XMLSchema/oval-common-5}note ).
```
As in more detailed way described here:
* http://making-security-measurable.1364806.n2.nabble.com/5-11-backwards-incompatibility-issue-Suspect-the-fix-for-issue-6-was-the-culprit-td7586190.html

when trying to use ```<notes>``` element in OVAL 5.11 leads to namespace mismatch between ```<notes>``` and its child ```<note>``` element.  AFAICT right now I can't see a way how to fix this in both OVAL 5.10 (which expects it to be in oval-def schema) and simultaneously in OVAL 5.11 (which expects it to be in oval-common-schema) in different way than to comment it out.

This needs to be fixed in OVAL standard (IOW the ```<notes>``` & ```<note>``` element either to be completely moved to ova-common-schema - both of them) or the 5.11 change to be reverted:
  https://github.com/OVALProject/Language/issues/237

If interested further, see the original 5.11 rc2 corresponding patch introducing that change:
  https://github.com/OVALProject/Language/commit/c34abb871ee446d9fd9e50a8ccbc8a8e2af5db20